examples/chip-tool/commands/common/Command.h

```diff
@@ -109,7 +109,7 @@ class Command
     Command(const char * commandName) : mName(commandName) {}
     virtual ~Command() {}
 
-    void SetExecutionContext(ExecutionContext & execContext) { mExecContext = &execContext; }
+    void SetExecutionContext(ExecutionContext & execContext) { memcpy(&mExecContext, &execContext, sizeof(execContext)); }
```

Comment:
Why this change?

Reply:
Explained in slack.

We need a deep copy of mExecContext.  Still, suggest this:

```suggestion
    void SetExecutionContext(ExecutionContext & execContext) { mExecContext = execContext; }
```